src/main/java/bouncybob/task/Deadlines.java

```diff
@@ -0,0 +1,58 @@
+package bouncybob.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task in the BouncyBob application.
+ */
+public class Deadlines extends Task {
```

Comment:
The Java coding standard encourages singular class names e.g. Deadline
```suggestion
public class Deadline extends Task {
```